src/modules/DNS.rb

```diff
@@ -479,6 +482,21 @@ def Import(settings)
 
       @nameservers = Builtins.eval(Ops.get_list(settings, "nameservers", []))
       @searchlist = Builtins.eval(Ops.get_list(settings, "searchlist", []))
+
+      # check for AY unsupported scenarios, the name servers and the search domains
+      # are written in the 2nd stage, if is disabled then it cannot work (bsc#1046198)
```

Comment:
I have the same opinion as Lada here. Just inform the user. But I would take a warning or info level.